force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -271,11 +347,16 @@ public with sharing class FlowControlLoopsCollections {
      */
     public static Integer findWordInList(String wordToFind, List<String> words) {
         // The variable to store the index of the word
+        Integer index = -1;
 
         // Loop through the list of words
-            // If the current word is the word to find, exit the loop
-
-        return null; // Replace null with the variable you used to store the result
+        for(Integer i=0; i< words.size(); i++){
+        // If the current word is the word to find, exit the loop
+            if(wordToFind == words.get(i)){
+                index = i;
+            }
+        }
+        return index; // Replace null with the variable you used to store the result
```

Comment:
Good. I would return from your loop as soon as you find that match. If you have a list with 1000000 members and you found the match on the first one, you don't want to continue checking. 

```
 public static Integer findWordInList(String wordToFind, List<String> words) {
     for(Integer i = 0; i < words.size(); i++){
         if(words[i] == wordToFind){
             return i;
         }
     }
        return -1;
    }
 ```